dart/test/http_client/failed_request_client_test.dart

```diff
@@ -253,13 +253,14 @@ class Fixture {
     bool sendDefaultPii = true,
   }) {
     final mc = client ?? getClient();
+    var hub = _hub;
+    hub.options.sendDefaultPii = sendDefaultPii;
     return FailedRequestClient(
       client: mc,
-      hub: _hub,
+      hub: hub,
```

Comment:
Not sure why you are doing this, `sendDefaultPii` should be removed from method params, and hub should take `_hub` and that's all.

Reply:
`fixture.getSut` was being called in two places with `sendDefaultPii: false`. This defaults to `true`. The tests were failling without doing this. I didn't want to change the values inside `_hub.options`.

Reply:
I have removed this